app/models/publisher.rb

```diff
@@ -346,6 +346,8 @@ def can_create_uphold_cards?
 
   # Remove when new dashboard is finished
   def in_new_ui_whitelist?
+    # for testing purposes :)
+    return true if self.email == "partner@brave.com" 
     self.email.in?((Rails.application.secrets[:new_ui_email_whitelist] || "").split(","))
```

Comment:
can `self.email` be modified client-side? ex: can i run client-side javascript that will set self.email to `partner@brave.com` ?